### ISSUE TITLE
Latest release v0.3.0 is over 200mb in size

### ISSUE THREAD

**Issue by n-i-x** [djm34/ccminer-msvc2015#36]
Seems like something is terribly wrong with the build. Can you verify that the size is supposed to be this big?

Also, it would be nice if md5/sha sums were included in releases.

**Comment by djm34** [comment]
nope nothing wrong... include nvidia support from 3.0 to 6.1
have no idea how to include checksum

**Comment by tamlin-mike** [comment]
_have no idea how to include checksum_

https://www.microsoft.com/en-us/download/details.aspx?id=11533

As for the size (problem), could the (rather thin, I'd hope) different CUDA wrappers be linked into different DLL's? Unless CUDA API changed massively, that could even allow for runtime selection of DLL/version to use, automatically and/or with command-line (override) option.

Additionally, if the binaries are linked to the static CRT (libc), it is possible to create binaries using directory-local msvcrt*.dll & co. to prevent messing with the _system_ (SxS and so on).

**Comment by djm34** [comment]
The idea was (and still is) a static build which can run almost anywhere and doesn't present any difficulty for new comers (and lazy people lol)  to the altcoin/mining scene.

I'll have a look for the checksum. thanks

**Comment by tamlin-mike** [comment]
_The idea was (and still is) a static build which can run almost anywhere_

Gotcha. That settles that (and I agree 100%).

**Comment by n-i-x** [comment]
I also agree. I just kinda freaked out when the size exploded without explanation. Maybe an update to the release notes are in order?

The whole checksum request was so that when things like this happen we can at least validate that it was intended.
